examples/nodejs/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM node:16 AS build-env
+FROM node:18 AS build-env
 COPY . /app
 WORKDIR /app
 
-RUN npm ci --only=production
+RUN npm ci --omit=dev
```

Comment:
Sorry, not super familiar with node, what's the difference here between these two options?

Reply:
When I was testing the changes, I got a warning in the console from NPM. Apparently `--only` [has been deprecated](https://docs.npmjs.com/cli/v8/using-npm/config#only).
![Screenshot 2022-05-09 at 21 11 48](https://user-images.githubusercontent.com/7257092/167489878-3dc31ed2-d7f1-40c3-aefa-28794f69d4be.png)

Reply:
sgtm